spark/src/main/java/org/apache/zeppelin/spark/dep/SparkDependencyContext.java

```diff
@@ -64,6 +67,8 @@ public SparkDependencyContext(String localRepoPath, String additionalRemoteRepos
   }
 
   public Dependency load(String lib) {
+    Console.println("DepInterpreter(%dep) deprecated. "
```

Comment:
I think printing message in the notebook is using Console is better in this case.